src/notebooks/controllers/kernelSource/notebookKernelSourceSelector.ts

```diff
@@ -236,6 +224,19 @@ export class NotebookKernelSourceSelector implements INotebookKernelSourceSelect
             });
         }
 
+        const localKernelFinder = allKernelFinders.find(
```

Comment:
Display local python env before localkernel specs (moved a block of code around)